deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -116,23 +116,25 @@ get_downloader() {
   fi
 }
 
-# The credentials can be specified either as already base64 encoded, or in plain text.
-# If the username or passphrase plain text variable is set but empty, the user will be asked for a value.
-if [ "${KIALI_USERNAME_BASE64}" == "" ]; then
-  KIALI_USERNAME="${KIALI_USERNAME=}" # note: the "=" inside ${} is on purpose
-  if [ "$KIALI_USERNAME" == "" ]; then
-    KIALI_USERNAME=$(read -p 'What do you want to use for the Kiali Username: ' val && echo -n $val)
+if [ "${AUTH_STRATEGY}" == "login" ]; then
```

Comment:
We are missing the strategy in the configmap - so this actually won't work if the user wants "anonymous" here.
There is no strategy setting here: https://github.com/kiali/kiali/blob/master/deploy/kubernetes/configmap.yaml

We need this to look like the openshift configmap so the installer can set the strategy.. see this: https://github.com/kiali/kiali/blob/v0.16.0/deploy/openshift/configmap.yaml#L19-L20